Networking/Networking/Model/OrderItem.swift

```diff
@@ -7,42 +7,51 @@ public struct OrderItem: Decodable {
     public let itemID: Int
     public let name: String
     public let productID: Int
-    public let quantity: NSDecimalNumber
+    public let variationID: Int
+    public let quantity: Decimal
```

Comment:
The API doc is misleading. It's documented as an `integer`, but if you add a decimal value from the web, it will accept it. This is especially common for shops that have weights or volumes for their quantities. You can order .25 ounces of liquids, .65 pounds fruit, .75 yards of fabric, for example.